app/components/common/eventModal/index.tsx

```diff
@@ -51,133 +63,164 @@ export default function EventModal({
-                      minDate={minDate.toDate()}
-                      selected={maxDate.toDate()}
-                      timeIntervals={5}
-                      excludeTimes={dateRange(
-                        dayjs(minDate).startOf('day').toDate(),
-                        minDate.toDate(),
-                      )}
-                      onChange={(end) => {
-                        if (minDate < dayjs(end)) {
-                          setEventDetail((e) => ({ ...e, end: dayjs(end).toDate() }));
-                        }
-                      }}
-                      showTimeSelect
-                      dateFormat="MMMM d, yyyy h:mm aa"
-                    />
-                  </div>
-
-                  <EmailChipsInput
-                    label="Attendees Emails"
-                    placeholder="Attendees Emails"
-                    attendees={eventDetail?.attendees ?? []}
-                    setAttendees={(attendees) => setEventDetail((ev) => ({ ...ev, attendees }))}
+                  <EventVisibility
+                    visibility={eventDetail?.visibility ?? 'private'}
+                    setVisibility={(visibility) => setEventDetail((e) => ({ ...e, visibility }))}
                   />
 
-                  <UserInput
-                    dataTestId="modal-location"
-                    label="URL / Address"
-                    labelClassnames="text-4"
-                    placeholder="Enter URL or Address for the event"
-                    value={eventDetail?.location ?? ''}
-                    setValue={(location) => setEventDetail((ev) => ({ ...ev, location }))}
-                  />
+                  <div className="mt-2">
+                    <div data-testid="modal-from-date">
+                      <p className="text-4 mb-2">From</p>
+                      <DatePicker
+                        disabled={text !== 'submit'}
+                        placeholderText="from-date"
+                        className="bg-stone-50 text-4 p-3 mb-6 focus:outline-none border border-solid border-stone-400 rounded-lg w-full cursor-pointer"
+                        selected={minDate.toDate()}
+                        maxDate={maxDate.toDate()}
+                        timeIntervals={5}
+                        excludeTimes={dateRange(
+                          maxDate.toDate(),
+                          dayjs(maxDate).endOf('day').toDate(),
+                        )}
+                        onChange={(start) => {
+                          if (dayjs(start) < maxDate) {
+                            setEventDetail((e) => ({ ...e, start: dayjs(start).toDate() }));
+                          }
+                        }}
+                        showTimeSelect
+                        dateFormat="MMMM d, yyyy h:mm aa"
+                      />
+                    </div>
+
+                    <div data-testid="modal-to-date">
+                      <p className="text-4 mb-2">To</p>
+                      <DatePicker
```

Comment:
Both component is using most of props which is similar, Can we store as props and stread so that we can avoid duplicate once

Reply:
can u share a similar example, i didn't get the whole comment